frontend/app/mixins/graphable.js

```diff
@@ -0,0 +1,94 @@
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
```

Comment:
@safeforge what do you think about `frontend/app/components/chart/mixins` subdir?